setup.py

```diff
@@ -30,7 +30,7 @@
     ],
     download_url="https://github.com/novoic/blabla/archive/v0.2.2.tar.gz",
     install_requires=[
-        "stanza==1.0.0",
+        "stanza>=1.1.0",
```

Comment:
We discussed pinning only major versions. What's the rationale here? Consider changing all to major version only